DigitalLearningSolutions.Data.Tests/DataServices/TutorialContentDataServiceTests/ObjectiveTests.cs

```diff
@@ -36,5 +36,69 @@ public void GetNonArchivedObjectivesBySectionAndCustomisationId_returns_objectiv
                 }
             }
         }
+
+        [Test]
+        public void GetNonArchivedCcObjectivesBySectionAndCustomisationId_returns_objectives_correctly()
+        {
+            using (new TransactionScope())
+            {
+                connection.Execute("UPDATE Tutorials SET OriginalTutorialID = 1 WHERE TutorialID = 1137");
+                connection.Execute("UPDATE CustomisationTutorials SET DiagStatus = 0 WHERE CusTutID = 704039");
+
+                // When
+                var result = tutorialContentDataService
+                    .GetNonArchivedCcObjectivesBySectionAndCustomisationId(248, 22062, false)
+                    .ToList();
+
+                // Then
+                using (new AssertionScope())
+                {
+                    result.Count.Should().Be(3);

```

Comment:
Because the PostLearning param is false, ideally the test would exclude some data that does not match the specific requirements. Do you think it is worth adding another assertion to check that the results do not include specific tutorial Ids that should be ignored?